R/tool_misc.R

```diff
@@ -128,16 +128,14 @@ twosls <- function(y, X, W, intercept = FALSE, lm.type = "lm"){
   
   if(intercept) {
     model <- switch(lm.type,
-                    "lm"      =  lm(y ~ Xhat),
-                    "lm.fit"  =  lm.fit(cbind(1, Xhat), y),
-                    ".lm.fit" = .lm.fit(cbind(1, Xhat), y))

```

Comment:
I wasn't sure how to weight ".lm.fit". I don't see the ".lm.fit" option being used, and this function isn't exported, so I just deleted it?